impl/src/main/java/org/eclipse/parsson/JsonMergePatchImpl.java

```diff
@@ -30,11 +31,14 @@
  * @since 1.1
  */
 
-public final class JsonMergePatchImpl implements JsonMergePatch {
+final class JsonMergePatchImpl implements JsonMergePatch {
```

Comment:
while not being exported on JPMS, it is still visible on classpath, so rather do not remove public here

Reply:
Yes, will keep public. But direct usage of anything in this package is wrong, users shall use jakarta.json API.